packages/button/src/Button/Button.spec.tsx

```diff
@@ -54,6 +54,31 @@ describe('packages/button', () => {
       expect(button.textContent).toBe(child);
     });
 
+    test(`renders spinner when isLoading is true`, () => {
```

Comment:
nit don't need backticks for these strings

Reply:
It's used throughout the file. Would we want to change all the backticks in the file now?